CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## 1.0.6
```

Comment:
Mark this as `## Next release` because many changes are in master now and i don't know what tag we will use for this